ui/src/components/WebRTCVideo.tsx

```diff
@@ -106,32 +106,57 @@ export default function WebRTCVideo() {
   );
 
   // Pointer lock and keyboard lock related
-  const isPointerLockPossible = window.location.protocol === "https:";
-
+  const isPointerLockPossible = window.location.protocol === "https:" || window.location.hostname === "localhost";
+  const isFullscreenEnabled = document.fullscreenEnabled;
```

Comment:
Let's have a single source of truth about _if_ we can offer/execute full-screen.